tasks/agent.py

```diff
@@ -276,8 +276,8 @@ def system_tests(_):
     pass
 
 
-@task
-def image_build(ctx, arch='amd64', base_dir="omnibus", python_version="2", skip_tests=False):
+@task(optional=['no_signed_pull'])
+def image_build(ctx, arch='amd64', base_dir="omnibus", python_version="2", skip_tests=False, no_signed_pull=False):
```

Comment:
Please flip the boolean to be `signed_pull=True`, otherwise we end up with a double negation `if not no_signed_pull`. Invoke already does the magic to understand the `no-` prefix, so you can actually pass `--no-signed-pull` even if the boolean is in positive.

Reply:
we wanted to have `signed_pull:true` by default.  that is why I changed the new to `--no-signed-pull`. 
What do you think If I renamed it: `--disable-signed-pull` ?

Reply:
if I define `signed_pull=True`, I got this error
```
➜ invoke  -e docker.pull-base-images Dockerfiles/agent/amd64/Dockerfile  --no-signed-pull
No idea what '--no-signed-pull' is!
```

Reply:
ok after removing `optional=['signed_pull']` it works.

Thanks for the great tip 👍

Reply:
@albertvaka ready for a new review :)